apps/condo/domains/meter/schema/CustomMeterName.js

```diff
@@ -0,0 +1,62 @@
+/**
+ * Generated by `createschema meter.CustomMeterName 'user:Relationship:User:CASCADE; meter:Relationship:Meter:CASCADE; name:Text'`
+ */
+
+const { historical, versioned, uuided, tracked, softDeleted, dvAndSender } = require('@open-condo/keystone/plugins')
+const { GQLListSchema } = require('@open-condo/keystone/schema')
+
+const access = require('@condo/domains/meter/access/CustomMeterName')
+const { RESIDENT } = require('@condo/domains/user/constants/common')
+
+
+const CustomMeterName = new GQLListSchema('CustomMeterName', {
```

Comment:
Maybe `MeterUserData` ?

Reply:
yes, i think it's more time proof incase we want to add any additional fields